<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <node pkg="catkin_template" type="cpp_node" name="cpp_node_instance" output="screen" respawn="false"/>
</launch>